<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Records</class>
 <widget class="QDialog" name="Records">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pontuação</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_1">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_3">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_4">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_5">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>200</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_6">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_7">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>240</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_8">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_9">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_10">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
